metabase/metabase
# Rework parameter mapping comparing to MLv2

Related to https://github.com/metabase/metabase/issues/38225

it's a proper fix instead of a workaround from https://github.com/metabase/metabase/pull/38513

### Description

Target is a `legacyRef`, so we can try to find a column from question instead of comparing targets

### How to verify

- you need to go to dashboard parameter mapping, choose some value and verify it's defined as "Unknown field"

### Demo

### Before
<img width="661" alt="image" src="https://github.com/metabase/metabase/assets/125459446/d50b2bd9-a5c2-47ef-bb07-79c11e458649">

After 

https://github.com/metabase/metabase/assets/125459446/856038fe-193a-40bb-bfbf-44da1a5e421a


### Checklist
